src/ImathTest/main.cpp

```diff
@@ -88,6 +89,11 @@ main (int argc, char* argv[])
     TEST (testFrustumTest);
     TEST (testInterop);
     TEST (testNoInterop);
+
```

Comment:
This needs a `#ifdef __CUDACC__`. That appears to be the reason the CI build fails.

Reply:
I think `__CUDACC__` is defined when nvcc compiles device code, so it never be defined in host code.

Reply:
That's the point, when compiling the code _without_ CUDA, it's currently failing because ``testVecCUDA`` isn't defined. Check the errors in the log from the failed CI job: https://github.com/AcademySoftwareFoundation/Imath/actions/runs/5599599937/jobs/10240779807?pr=334#step:6:105
Thanks!

Reply:
I fixed. I'm sorry, but could you please review?